frontend/Store.js

```diff
@@ -150,6 +151,7 @@ class Store extends EventEmitter {
     this._establishConnection();
     this._eventQueue = [];
     this._eventTimer = null;
+    this._emitSearchRootsChange = debounce(this.filterSearchRoots, 100);
```

Comment:
Got it. Cool because that means we might get this thing merged soon :simple_smile: